src/main/java/juja/microservices/keepers/dao/KeepersRepository.java

```diff
@@ -42,34 +44,44 @@
     public String save(KeeperRequest keeperRequest) {
         Date startDate = Date.from(LocalDateTime.now().atZone(ZoneId.systemDefault()).toInstant());
         Keeper keeper = new Keeper(keeperRequest.getFrom(),
```

Comment:
@prfct It would be easier you fix it, because of possible conflicts. And you know what to do.